src/EnergyPlus/CostEstimateManager.cc

```diff
@@ -683,6 +694,85 @@ namespace CostEstimateManager {
+                                Qty += e.performance.normalMode.ratedGrossTotalCap;
+                            state.dataCostEstimateManager->CostLineItem(Item).Qty = Qty / 1000.0;
+                            state.dataCostEstimateManager->CostLineItem(Item).Units = "kW (tot cool cap.)";
+                            state.dataCostEstimateManager->CostLineItem(Item).ValuePer =
+                                state.dataCostEstimateManager->CostLineItem(Item).PerKiloWattCap;
+                            state.dataCostEstimateManager->CostLineItem(Item).LineSubTotal =
+                                state.dataCostEstimateManager->CostLineItem(Item).Qty * state.dataCostEstimateManager->CostLineItem(Item).ValuePer;
+                        }
+                        if (coilFound) {
+                            state.dataCostEstimateManager->CostLineItem(Item).Qty =
+                                state.dataCoilCooingDX->coilCoolingDXs[thisCoil].performance.normalMode.ratedGrossTotalCap / 1000.0;
+                            state.dataCostEstimateManager->CostLineItem(Item).Units = "kW (tot cool cap.)";
+                            state.dataCostEstimateManager->CostLineItem(Item).ValuePer =
+                                state.dataCostEstimateManager->CostLineItem(Item).PerKiloWattCap;
+                            state.dataCostEstimateManager->CostLineItem(Item).LineSubTotal =
+                                state.dataCostEstimateManager->CostLineItem(Item).Qty * state.dataCostEstimateManager->CostLineItem(Item).ValuePer;
+                        }
+                    }
+
+                    if (state.dataCostEstimateManager->CostLineItem(Item).PerEach > 0.0) {
+                        if (WildcardObjNames)
+                            state.dataCostEstimateManager->CostLineItem(Item).Qty = double(state.dataCoilCooingDX->coilCoolingDXs.size());
+                        if (thisCoil > 0) state.dataCostEstimateManager->CostLineItem(Item).Qty = 1.0;
+                        state.dataCostEstimateManager->CostLineItem(Item).ValuePer = state.dataCostEstimateManager->CostLineItem(Item).PerEach;
+                        state.dataCostEstimateManager->CostLineItem(Item).LineSubTotal =
+                            state.dataCostEstimateManager->CostLineItem(Item).Qty * state.dataCostEstimateManager->CostLineItem(Item).ValuePer;
+                        state.dataCostEstimateManager->CostLineItem(Item).Units = "Ea.";
+                    }
+
+                    if (state.dataCostEstimateManager->CostLineItem(Item).PerKWCapPerCOP > 0.0) {
+                        if (WildcardObjNames) {
+                            Real64 Qty(0.0);
+                            for (auto const &e : state.dataCoilCooingDX->coilCoolingDXs) {
+                                // TODO: Using 0 index to match above Coil:DX object
+                                Real64 COP = e.performance.normalMode.speeds[0].original_input_specs.gross_rated_cooling_COP;
```

Comment:
Is 0 the highest speed? I think I would use numSpeeds here. Not sure what to do for VS coils. For a cost estimate it seems reasonable to use the rated COP and TotCap at highest speed.

Reply:
Coil:DX in the previous block always uses the first speed. i.e ```DXCoil(thisCoil).RatedTotCap(1)```.

That doesn't mean that we should continue with that precedent. I'll defer to your recommendations.

Reply:
That seems wrong. It should be full capacity. But for this coil, that comes from the performance object, I think.

Reply:
Do we need to address this change then?  Mine out the full capacity based on nominal speed or whatever it is called?  Otherwise this coding looks good.  It would be great to go ahead and add a unit test or at least an example file with this in place.

Reply:
Following up @mjwitte @rraustad @Myoldmopar 

Do we fix the old coil to use the highest speed as part of this?

Reply:
Yes. If you haven't yet, please post an issue to note that it wasn't using the full capacity before.

Reply:
I agree. This needs to be fixed for any existing VS coil models.